examples/resources/custom_domain/main.tf

```diff
@@ -0,0 +1,50 @@
+terraform {
+  required_providers {
+    propelauth = {
+      source = "registry.terraform.io/propelauth/propelauth"
+    }
+    # aws = {
+    #   source  = "hashicorp/aws"
+    #   version = "~> 5.0"
+    # }
```

Comment:
let's clean up this example and it's comments, so it provides a clear guide on how to us our provider w/ a dns provider

Reply:
I'll own cleaning this up and formatting it for docs generation in [this adjacent PR](https://github.com/PropelAuth/terraform-provider-propelauth/pull/7/files).